includes/onnxruntime_customop.hpp

```diff
@@ -295,3 +295,7 @@ inline OrtValue* CustomOpApi::KernelContext_GetOutput(OrtKernelContext* context,
 }
 
 }  // namespace OrtW
+
+// !! TODO: only do it for legecy ort build
```

Comment:
I guess the logic should be implemented into CMakeLists.txt?

Reply:
yes let's fix it later once the API is finalized.